README.md

```diff
@@ -18,8 +18,6 @@ The template is a working application with a minimal setup. It contains:
  * swagger configuration for api documentation ([see how to publish your api documentation to shared repository](https://github.com/hmcts/reform-api-docs#publish-swagger-docs))
  * code quality tools already set up
  * integration with Travis CI
- * Hystrix circuit breaker enabled
```

Comment:
Every microservices should start implementing hystrix circuit breaker to make the benefit of REST exchange. Can we leave it in template?